python/lsst/daf/butler/registry/dimensions/static.py

```diff
@@ -50,6 +53,33 @@
 _VERSION = VersionTuple(5, 0, 0)
 
 
+def _makeDimensionGraphTableSpecs() -> ddl.TableSpec:
```

Comment:
Is this still used?

Reply:
I thought I'd deleted that.  I certainly will now.